game-core/src/main/java/org/triplea/game/server/HeadlessGameServer.java

```diff
@@ -607,79 +586,41 @@ private static void usage() {
-        + "   " + LOBBY_GAME_HOSTED_BY + "=<LOBBY_GAME_HOSTED_BY>\n"
-        + "   " + LOBBY_GAME_SUPPORT_EMAIL + "=<youremail@emailprovider.com>\n"
         + "   " + LOBBY_GAME_SUPPORT_PASSWORD + "=<password for remote actions, such as remote stop game>\n"
-        + "   " + LOBBY_GAME_RECONNECTION + "=<seconds between refreshing lobby connection [min "
-        + LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM + "]>\n"
-        + "   " + MAP_FOLDER + "=mapFolder"
-        + "\n"
-        + "   You must start the Name and HostedBy with \"Bot\".\n"
-        + "   Game Comments must have this string in it: \"automated_host\".\n"
-        + "   You must include a support email for your host, so that you can be alerted by lobby admins when your "
-        + "host has an error."
-        + " (For example they may email you when your host is down and needs to be restarted.)\n"
-        + "   Support password is a remote access password that will allow lobby admins to remotely take the "
-        + "following actions: ban player, stop game, shutdown server."
-        + " (Please email this password to one of the lobby moderators, or private message an admin on the "
-        + "forums.triplea-game.org forum.)\n");
+        + "   " + MAP_FOLDER + "=<MAP_FOLDER>"
+        + "\n");
   }
 
   private static void handleHeadlessGameServerArgs() {
     boolean printUsage = false;
 
-    final File mapFolder = ClientSetting.mapFolderOverride.getValueOrThrow().toFile();
-    if (!mapFolder.isDirectory()) {
-      log.warning("Invalid '" + MAP_FOLDER + "' param, map folder must exist: '" + mapFolder + "'");
-      printUsage = true;
+    if (!ClientSetting.mapFolderOverride.isSet()) {
+      ClientSetting.mapFolderOverride.setValue(
+          Paths.get(SystemProperties.getUserHome(), "/triplea/downloadedMaps"));
     }
 
     final String playerName = System.getProperty(TRIPLEA_NAME, "");
-    final String hostName = System.getProperty(LOBBY_GAME_HOSTED_BY, "");
-    if (playerName.length() < 7 || hostName.length() < 7 || !hostName.equals(playerName)
-        || !playerName.startsWith("Bot") || !hostName.startsWith("Bot")) {
-      log.warning(
-          "Invalid argument: " + TRIPLEA_NAME + " and " + LOBBY_GAME_HOSTED_BY
-              + " must start with \"Bot\" and be at least 7 characters long and be the same.");
+    if ((playerName.length() < 7) || !playerName.startsWith("Bot")) {
+      log.warning("Invalid argument: " + TRIPLEA_NAME + " must at least 7 characters long");
       printUsage = true;
     }
 
-    final String comments = System.getProperty(LOBBY_GAME_COMMENTS, "");
-    if (!comments.contains("automated_host")) {
-      log.warning(
-          "Invalid argument: " + LOBBY_GAME_COMMENTS + " must contain the string \"automated_host\".");
+    if (Integer.valueOf(System.getProperty(TRIPLEA_PORT, "0")) <= 0) {
+      log.warning("Invalid argument: " + TRIPLEA_PORT + " must be numeric");
```

Comment:
```suggestion
      log.warning("Invalid argument: " + TRIPLEA_PORT + " must be greater than zero");
```

If you want to check for non-numeric values, you're going to have to catch `NumberFormatException`.

Reply:
Fixed in: 198abcf57